src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -936,7 +936,9 @@ void program_node::init_onednn_primitive_attributes() {
             } else if (fused_desc->activation_function == cldnn::activation_func::hsigmoid) {
                 // hard_sigmoid(x,a,b) = clamp(ax+b, 0, 1)
                 // hsigmoid(x) = clamp(val+3, 0, 6) / 6 = clamp(val/6+0.5, 0, 1) = hard_sigmoid(val, 1/6, 1/2)
-                post_ops.append_eltwise(dnnl::algorithm::eltwise_hardsigmoid, 1./6, 1./2);
+                post_ops.append_eltwise(dnnl::algorithm::eltwise_hardsigmoid,
+                                        static_cast<float>(1. / 6),
+                                        static_cast<float>(1. / 2));
```

Comment:
why not 1.0f ?

Reply:
@panhaiqi Please take a look, it should help to avoid static_cast.